ui/static/ui/js/learning_resources.jsx

```diff
@@ -478,15 +532,18 @@ define('learning_resources',
     LearningResourcePanel: LearningResourcePanel,
     loader: function (
       repoSlug, learningResourceId, refreshFromAPI,
-      closeLearningResourcePanel, container
+      closeLearningResourcePanel, closeLRPanel, isNewResource, container
     ) {
-      // Unmount and remount the component to ensure that its state
-      // is always up to date with the rest of the app.
-      React.unmountComponentAtNode(container);
+      if (isNewResource) {
+        // Unmount and remount the component to ensure that its state
+        // is always up to date with the rest of the app.
+        React.unmountComponentAtNode(container);
+      }
       React.render(<LearningResourcePanel
         repoSlug={repoSlug}
         learningResourceId={learningResourceId}
         refreshFromAPI={refreshFromAPI}
+        closeLRPanel={closeLRPanel}
```

Comment:
Can you rename `closeLRPanel` to `requestedPanelClose` everywhere in this file? Otherwise it looks too similar to `closeLearningResourcePanel`